composer.json

```diff
@@ -52,6 +52,9 @@
 		"check-cs": [
 			"\"vendor/bin/phpcs\""
 		],
+		"check-cs-errors": [
+			"\"vendor/bin/phpcs\" --error-severity=1 --warning-severity=6"
```

Comment:
I just needed to have a quick command in my terminal to only see the errors while checking, this is (not yet) used in the build process)